es2020.md

```diff
@@ -41,3 +41,16 @@ interface ImportExpression <: Expression {
   node, but it can be an arbitrary expression node.
 
 [ImportExpression]: es2015.md#importdeclaration
+
+# Modules
+
+##  ExportAllDeclaration
+
+```js
+interface ExportAllDeclaration <: ModuleDeclaration {
```

Comment:
If it's extending an existing interface, this should say `extend interface`.

Reply:
I've corrected the code as you said